<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>637</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>1400</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="maximumSize">
          <size>
           <width>1400</width>
           <height>120</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.55 rgba(61, 235, 235, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; font-style:italic; color:#ffffff;&quot;&gt;                                                  INDUSTRIAL MANAGEMENT SYSTEM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>1500</width>
        <height>1400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 241, 227);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>421</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>1400</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(221, 219, 203);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>150</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../../../final projext/240_F_103293921_HnSUxsEatUrgvdOp57IuriUeMrpRlMrt-removebg-preview.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
  
	background-color: rgb(27, 154, 170);
	color:rgb(0,0,0);
	border-radius:10%;
}

QPushButton:hover{
	
	background-color: rgb(245, 241, 227);
	
	color: rgb(0, 0, 0);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Add Vehical</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
  
	background-color: rgb(27, 154, 170);
	color:rgb(0,0,0);
	border-radius:10%;
}

QPushButton:hover{
	
	background-color: rgb(245, 241, 227);
	
	color: rgb(0, 0, 0);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Edit Vehical</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
  
	background-color: rgb(27, 154, 170);
	color:rgb(0,0,0);
	border-radius:10%;
}

QPushButton:hover{
	
	background-color: rgb(245, 241, 227);
	
	color: rgb(0, 0, 0);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
  
	background-color: rgb(27, 154, 170);
	color:rgb(0,0,0);
	border-radius:10%;
}

QPushButton:hover{
	
	background-color: rgb(245, 241, 227);
	
	color: rgb(0, 0, 0);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
  
	background-color: rgb(27, 154, 170);
	color:rgb(0,0,0);
	border-radius:10%;
}

QPushButton:hover{
	
	background-color: rgb(245, 241, 227);
	
	color: rgb(0, 0, 0);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Complain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
  
	background-color: rgb(27, 154, 170);
	color:rgb(0,0,0);
	border-radius:10%;
}

QPushButton:hover{
	
	background-color: rgb(245, 241, 227);
	
	color: rgb(0, 0, 0);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>View</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
  
	background-color: rgb(27, 154, 170);
	color:rgb(0,0,0);
	border-radius:10%;
}

QPushButton:hover{
	
	background-color: rgb(245, 241, 227);
	
	color: rgb(0, 0, 0);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>franchise</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
  
	background-color: rgb(27, 154, 170);
	color:rgb(0,0,0);
	border-radius:10%;
}

QPushButton:hover{
	
	background-color: rgb(245, 241, 227);
	
	color: rgb(0, 0, 0);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Orders</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_5" native="true">
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>400</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 85, 255);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(27, 154, 170);
color:rgb(221, 219, 203);
font-family:times new roman;


</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Order Summary&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <property name="styleSheet">
             <string notr="true">selection-background-color: rgb(59, 228, 228);</string>
            </property>
            <column>
             <property name="text">
              <string>name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>price</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>city</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>model</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>cc</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>mode</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
